po/es.po

```diff
@@ -11398,15 +11400,15 @@ msgstr ""
 #: ../src/iop/vignette.c:1193
 msgid "enable to have the ratio automatically follow the image size"
 msgstr ""
-"activar para que la proporción siga el tamaño de imagen automáticamente"
+"Activar para que la proporción siga automáticamente el tamaño de imagen"
 
 #: ../src/iop/vignette.c:1194
 msgid "width-to-height ratio"
-msgstr "proporción ancho/alto"
+msgstr "Factor de proporción entre la altura y la anchura"
```

Comment:
I think, original translation was more clear and **concise**.

Reply:
Not really, or I would have left it as it was. You have to use the modules.